### PR TITLE
Remove EaterEvals link

### PR DESCRIPTION
**Pull request by ChaseC99**
## Summary
The url for EaterEvals changed, so we are currently unable to link to it.
For now, we are going to remove EaterEvals and just send users to RMP.

Other changes in this commit
- Update RMP url to include UCI's school id
- Fix a compiler warning for "Duplicate key 'sectionCode'"

## Test Plan
Pull up a class and verify that clicking the professor's name brings the user to RMP.

## Issues
References #174 

## Future Followup
Consider adding EaterEvals back if we can map the instructor's last name to their uci net id.

### CODE REVIEW

File: client/src/components/SectionTable/SectionTableBody.js [hunk 1]
```diff
@@ -67,11 +57,6 @@ const styles = (theme) => ({
     full: {
         color: '#e53935',
     },
-    sectionCode: {
-        '&:hover': {
-            cursor: 'pointer',
-        },
-    },
```
Comment (ChaseC99) [inline client/src/components/SectionTable/SectionTableBody.js:74]: Duplicate key for `sectionCode`. Added this property to the already existing one above.

File: client/src/components/SectionTable/SectionTableBody.js [hunk 2]
```diff
@@ -171,60 +156,27 @@ const SectionDetailsCell = withStyles(styles)((props) => {
...
-                        key={profName}
-                        interactive
-                        placement="left"
-                        title={<DualButton profName={profName} />}
-                    >
-                        <div className={classes.link}>{profName}</div>
-                    </CustomTooltip>
+                    <div>
+                        <a
+                            href={`https://www.ratemyprofessors.com/search/teachers?sid=U2Nob29sLTEwNzQ=&query=${lastName}`}
```
Comment (ChaseC99) [inline client/src/components/SectionTable/SectionTableBody.js:166]: Updated the URL to include UCI's school id. This provides better search results than the previous url.

File: client/src/components/SectionTable/SectionTableBody.js [hunk 3]
```diff
@@ -366,13 +318,17 @@ const SectionTableBody = withStyles(styles)((props) => {
...
+        const toggleHighlight = () => {
+            if (
+                AppStore.getAddedSectionCodes()[AppStore.getCurrentScheduleIndex()].has(
+                    `${section.sectionCode} ${term}`
+                )
+            )
+                setAddedCourse(true);
+            else setAddedCourse(false);
+        };
+
```
Comment (ChaseC99) [inline client/src/components/SectionTable/SectionTableBody.js:331]: Formatting auto-updated by the linter.

**Comment by devsdevsdevs** [comment]
Looks good, merging for now but keep branch open to resolve #174

**Pull request merged**